docs/wiki/end-user-guide/preparing-your-modpack.md

```diff
@@ -0,0 +1,91 @@
+| **LibrarianLib**                | Will not work correctly with Forgelin-Continuous            | :white_check_mark: Use **[LibrarianLib-Continuous](https://www.curseforge.com/minecraft/mc-mods/librarianlib-continuous)** (May break Wizardry Animations)                                                  |
+| **Mouse Tweaks**                | Outdated and prone to cause issues                          | :white_check_mark: Use **[Mouse Tweaks Unofficial](https://www.curseforge.com/minecraft/mc-mods/mouse-tweaks-unofficial)**                                                                                  |
+| **Phosphor**                    | Incompatible                                                | :white_check_mark: Use **[Alfheim Lighting Engine](https://www.curseforge.com/minecraft/mc-mods/alfheim-lighting-engine)** / **[Hesperus](https://www.curseforge.com/minecraft/mc-mods/hesperus)**          |
+| **Polyfrost series**            | Awaiting official fixes.                                    | —                                                                                                                                                                                                           |
+| **Raw Input**                   | Already included in Cleanroom.                              | :no_entry_sign: Remove                                                                                                                                                                                      |
+| **SerializationIsBad**          | Redundant. Modern Java versions fix the issue this solves   | :no_entry_sign: Remove                                                                                                                                                                                      |
+| **TexFix**                      | Merged and rewritten in CensoredASM                         | :white_check_mark: Use **[CensoredASM](https://www.curseforge.com/minecraft/mc-mods/lolasm)**                                                                                                               |
+| **VanillaFix**                  | Merged and improved in CensoredASM                          | :white_check_mark: Use **[CensoredASM](https://www.curseforge.com/minecraft/mc-mods/lolasm)**                                                                                                               |
+| **Secret Rooms**                | Has an issue with updater                                   | :no_entry_sign: Disable the update checker in config file                                                                                                                                                   |
+
+::: info Note {id="note"}
+If your modpack uses **Bansoukou**, make sure to **run the instance at least once** before installing cleanroom loader so its patches are applied.
+:::
+## Performance / Memory optimisation mods
```

Comment:
`optimisation` is the british spelling of `optimization` - i believe we generally use american english (ie `color` not `colour`), so that should be changed